easybuild/easyconfigs/p/phyx/phyx-1.3-foss-2020b.eb

```diff
@@ -0,0 +1,49 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+easyblock = 'ConfigureMake'
+
+name = 'phyx'
+version = '1.3'
+
+homepage = 'https://github.com/FePhyFoFum/%(name)s'
+description = "phyx performs phylogenetics analyses on trees and sequences."
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'pic': True}
+
+github_account = 'FePhyFoFum'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+patches = ['%(name)s-1.01_link_openblas.patch']
+checksums = [
+    'b09677f98f997838b39cd2724463be06d0627ddc90a7e659dc90d4e02db624eb',  # v1.3.tar.gz
+    '8c43454c7ad87cfa9ac1eacef31fb651637f8ae218d6337beabbc98899eb574d',  # phyx-1.01_link_openblas.patch
+]
+
+builddependencies = [
+    ('Autotools', '20200321', '', ('GCCcore', '10.2.0')),
+]
+
+dependencies = [
+    ('Armadillo', '10.5.3'),
+    ('NLopt', '2.6.2'),
+]
+
+start_dir = 'src'
+
+preconfigopts = "autoconf && "
+
+preinstallopts = 'mkdir %(installdir)s/bin && '
+
+sanity_check_paths = {
+    'files': ['bin/' + _exec for _exec in ["pxaa2cdn", "pxbdfit", "pxbdsim",
+                                           "pxboot", "pxbp", "pxcat", "pxclsq", "pxcltr", "pxcolt", "pxconsq",
+                                           "pxcontrates", "pxfqfilt", "pxlog", "pxlssq", "pxlstr", "pxmrca",
+                                           "pxmrcacut", "pxmrcaname", "pxnw", "pxrecode", "pxrevcomp",
+                                           "pxrls", "pxrlt", "pxrmk", "pxrms", "pxrmt", "pxrr", "pxs2fa", "pxs2nex",
+                                           "pxs2phy", "pxseqgen", "pxssort", "pxsstat", "pxstrec", "pxsw", "pxt2new",
+                                           "pxt2nex", "pxtcol", "pxtcomb", "pxtlate", "pxtrt", "pxtscale",
+                                           "pxvcf2fa"]],
+    'dirs': [],
+}
+
```

Comment:
can you add a few `sanity_check_commands` here? 
```
sanity_check_commands = [
    'pxaa2cdn --help',
    ...
]
```

Reply:
I'm not a user of that program myself. So, at most I could add stuff like "pfoobar --help". But is that worthwhile one the existence of the binaries is already ensured?

Reply:
the sanity check commands are meant for a minimal check of the binaries: catching missing linked libraries, etc.
fully testing their correctness is out of scope here.
adding `pfoobar --help` for a couple of binaries is sufficient.